GrandCentralBoard/Widgets/Harvest/HarvestWidgetView.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  HarvestWidgetView.swift
+//  GrandCentralBoard
+//
+//  Created by Karol Kozub on 2016-04-11.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import UIKit
+
+
+class HarvestWidgetView: UIView {
```

Comment:
View should be as far from what it presets as possible. The view should be called `SimpleBarChart` or something like that. To describe how it presents data, not what data it presents.

But I can change it in the future.